tests/MyGame/Example/Monster.cs

```diff
@@ -40,8 +40,8 @@ public sealed class Monster : Table {
   public Monster GetTestarrayoftables(int j) { return GetTestarrayoftables(new Monster(), j); }
   public Monster GetTestarrayoftables(Monster obj, int j) { int o = __offset(26); return o != 0 ? obj.__init(__indirect(__vector(o) + j * 4), bb) : null; }
   public int TestarrayoftablesLength { get { int o = __offset(26); return o != 0 ? __vector_len(o) : 0; } }
-  public Monster Enemy { get { return GetEnemy(new Monster()); } }
-  public Monster GetEnemy(Monster obj) { int o = __offset(28); return o != 0 ? obj.__init(__indirect(o + bb_pos), bb) : null; }
+  public MyGame.Example2.Monster Enemy { get { return GetEnemy(new MyGame.Example2.Monster()); } }
```

Comment:
I ran the generate_code.sh script and it generated all these changes. I'll remove them from patch.